### PR TITLE
Update the README.md to describe the repo

### PR DESCRIPTION
**Pull request by SunWithIssues**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,88 @@
...
     <script src="script.js"></script>
   </head>
   <body>
+    
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <!--Header  -->
+      <h1 style="margin:0;padding:0; font-family:'Brush Script MT'; font-size:80px">
```
Comment (emwarman) [inline portfolio/src/main/webapp/index.html]: What do you think about adding inline styles like this to your stylesheet?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,88 @@
...
+      <h2 style="margin:0;padding:0">A portfolio made for SPS2020</h2>
+      <hr>
+
+      
+      <p style="margin-top:30px">Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      
+      <!-- Intro: details --> 
+      <p>My name is Anya Ellis. I am an undergrad at the University of Illinois at Chicago (UIC).
```
Comment (emwarman) [inline portfolio/src/main/webapp/index.html:25]: I didn't know you went to UIC! I graduated from IIT a little further southeast!

**Review by emwarman — CHANGES_REQUESTED** [review]
Looks great, Anya! Loved getting to know you more through your portfolio.

I had one small comment before submitting.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (SunWithIssues) [reply portfolio/src/main/webapp/index.html]: Oh! That makes sense. I created a new `<div>` tag as well. Hopefully this makes it more readable.

`#banner h1 {                               `
`  margin:0;                                   `
`  padding:0;                                 `
`  font-family:'Brush Script MT';    `
`  font-size:80px;                           `
`}                                                   `

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (SunWithIssues) [reply portfolio/src/main/webapp/index.html:25]: Yeah, I know a couple of people from IIT from conferences but I haven't visited the campus yet. I mostly just hang out in Pilsen and Little Italy.

**Review by emwarman — APPROVED** [review]
LGTM!

**Pull request merged**